### PR TITLE
Add strictly typed responses; fix circular dependencies; improve namespaces

### PR DESCRIPTION
**Pull request by dnl-blkv**

### CODE REVIEW

File: bunq/sdk/json/adapters.py [hunk 1]
```diff
@@ -29,25 +29,25 @@ class InstallationAdapter(converter.JsonAdapter):
     @classmethod
     def deserialize(cls, target_class, array):
         """
-        :type target_class: model.Installation|type
+        :type target_class: core.Installation|type
```
Comment (OGKevin) [inline bunq/sdk/json/adapters.py]: Is this notation used every where else in this SDK as well ? 🙊

File: bunq/__init__.py [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+from bunq.sdk.json import converter
 
-registry.initialize()
+
+def initialize_converter():
+    """
+    :rtype: None
+    """
+
+    import datetime
```
[the remark this answers is not in the document]
Comment (dnl-blkv) [reply bunq/__init__.py]: It is an exception made for the initializer to make sure no code from within the module is executed before it is needed. Reducing chances of circular deps!

File: bunq/sdk/json/adapters.py (hunk 1 above)
Comment (dnl-blkv) [reply bunq/sdk/json/adapters.py]: Notation, as in `|type`? Yes, absolutely! For long time already :)
Same goes for `module.Class`! 👍

**Comment by dnl-blkv** [comment]
@OGKevin all questions answered
@andrederoos please take a look :D

**Pull request merged**